doc/tutorial/ranges/index.md

```diff
@@ -175,7 +175,7 @@ It depends on the underlying range and also the view itself.
 With few exceptions, views don't model more/stronger range concepts than their underlying range (other than
 std::ranges::view) and they try to preserve as much of the underlying range's concepts as possible.
 For instance the view returned by `std::views::reverse` models std::ranges::random_access_range (and weaker concepts)
-iff the underlying range also models the respective concept.
+if the underlying range also models the respective concept.
```

Comment:
I don't know if you are aware of `iff` (https://en.wikipedia.org/wiki/If_and_only_if)
but I think you are right that in this case `iff` does not apply because the former statement is always true regardless of the latter.